### PR TITLE
Added blogs for GSoC 2021.

### PR DESCRIPTION
**Pull request by AGaliciaMartinez**
Added links for GSoC blogs.

### CODE REVIEW

**Comment by nathanshammah** [comment]
Thanks @AGaliciaMartinez. @purva-thakre, can we add yours too?

**Comment by nathanshammah** [comment]
If having the blog up takes longer, we should at least open a PR with a news item about the new GSoC students and project.s

**Comment by purva-thakre** [comment]
@AGaliciaMartinez @nathanshammah  You can add it. Here's a link to mine : https://purva-thakre.github.io/purva-blog/ 
I am still making changes to the layout etc. for my blog. Thanks !

File: news.html [hunk 1]
```diff
@@ -11,6 +11,21 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
...
+
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>June 7, 2021</p>
+        <h4>2021 Google Summer of Code Project Updates
+</h4>
+    </div>
+<div class="col-md-4" style='margin-top: 25px'>
+        <a href="https://felipebihaiek.medium.com/"><p>Felipe's Blog posts</p></a>
+        <a href=""><p>Purva's Blog posts</p></a>
```
Comment (purva-thakre) [inline news.html:24]: https://purva-thakre.github.io/purva-blog/

Comment (AGaliciaMartinez) [reply news.html:24]: Thank you @purva-thakre!

File: news.html [hunk 2]
```diff
@@ -11,6 +11,21 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
...
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>June 7, 2021</p>
+        <h4>2021 Google Summer of Code Project Updates
+</h4>
+    </div>
+<div class="col-md-4" style='margin-top: 25px'>
+        <a href="https://felipebihaiek.medium.com/"><p>Felipe's Blog posts</p></a>
+        <a href="https://purva-thakre.github.io/purva-blog/"><p>Purva's Blog posts</p></a>
+        <a href="https://agalicia.netlify.app/"><p>Asier's Blog Posts</p></a>
```
Comment (nathanshammah) [inline news.html:25]:
```suggestion
        <a href="https://felipebihaiek.medium.com/"><p>Felipe's blog posts</p></a>
        <a href="https://purva-thakre.github.io/purva-blog/"><p>Purva's blog posts</p></a>
        <a href="https://agalicia.netlify.app/tag/gsoc/"><p>Asier's Blog Posts</p></a>
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]

File: news.html (hunk 2 above)
Comment (nathanshammah) [reply news.html:25]: Also Asier's B-->b.

**Comment by nathanshammah** [comment]
@purva, I noticed in your blog: QuTip-->QuTiP.

**Comment by purva-thakre** [comment]
> @Purva, I noticed in your blog: QuTip-->QuTiP.

Thanks ! Should be fixed now.

**Comment by purva-thakre** [comment]
@nathanshammah I have made enough changes to my blog for now. 

There are a couple issues which I hope to work on later as I work on trying to understand `Jekyll`. For example, there's no way to add a comment to my blog posts but I have made a note about this at the bottom of my first post.

**Review by nathanshammah — APPROVED** [review]
Looks great, thanks @AGaliciaMartinez! @purva-thakre: looks like a nice first blog post. Comments are not required as a feature.

**Pull request merged**